src/web/js/trove/chart-lib.js

```diff
@@ -827,17 +1218,24 @@ ${labelRow}`;
     });
   }
 
-  return RUNTIME.makeObject({
-    'provide-plus-types': RUNTIME.makeObject({
-      types: RUNTIME.makeObject({}),
-      values: RUNTIME.makeObject({
-        'pie-chart': makeFunction(pieChart),
-        'bar-chart': makeFunction(barChart),
-        'histogram': makeFunction(histogram),
-        'box-plot': makeFunction(boxPlot),
-        'plot': makeFunction(plot),
-      })
-    })
-  });
+  return RUNTIME.makeModuleReturn(
+    {
+      'pie-chart': makeFunction(pieChart),
+      'bar-chart': makeFunction(barChart),
+      'multi-bar-chart': makeFunction(multiBarChart),
+      'histogram': makeFunction(histogram),
+      'box-plot': makeFunction(boxPlot),
+      'plot': makeFunction(plot),
+    }, 
+    {
+      "LoC": ann("List<Color>", checkListWith(IMAGE.isColorOrColorString)),
+      "LoS": ann("List<String>", checkListWith(RUNTIME.isString)), 
+      "LoN": ann("List<Number>", checkListWith(RUNTIME.isNumber)),
+      "LoLoN": ann("List<List<Number>>", checkListWith(checkListWith(RUNTIME.isNumber))),
+      "LoLoLoN": ann("List<List<List<<Number>>>", checkListWith(checkListWith(checkListWith(RUNTIME.isNumber)))),
```

Comment:
Is the idea that the user will use these types?

Reply:
The main idea behind this was that pyret doesn't typecheck nested types so many of these methods that would require List<Type> or Option<Type> Etc... would give fairly unhelpful type errors when the wrong nested type was given. This is just so that we can give better type errors in that edge case!